components/automate-ui/src/app/components/chef-components.module.ts

```diff
@@ -111,6 +125,8 @@ import { TableHeaderComponent } from './table/table-header/table-header.componen
     BreadcrumbComponent,
     CalendarComponent,
     ChefNotificationsComponent,
+    ConfirmApplyStartModalComponent,
```

Comment:
@bcmdarroch so, these components are modals and made for the projects page. They were located in the projects folder structure and added to the projects module. Which means they were not accessible to the applications components. I moved them into the components folder and added them to the chef components module so that they could be used in the bottom bars.